<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="ZCBLRLP0293" failures="1" tests="19" name="zapcg.Capillary.EarnPointPageTestCases.EarnPoint_Valid_TestCases" time="224.414" errors="1" timestamp="2024-07-06T15:43:26 IST" skipped="0">
  <testcase classname="zapcg.Capillary.EarnPointPageTestCases.EarnPoint_Valid_TestCases" name="verifyNonEditableLocationField" time="9.986"/>
  <testcase classname="zapcg.Capillary.EarnPointPageTestCases.EarnPoint_Valid_TestCases" name="verifyNonEditableDateField" time="7.483"/>
  <testcase classname="zapcg.Capillary.EarnPointPageTestCases.EarnPoint_Valid_TestCases" name="verifyNonEditableTimeField" time="8.044"/>
  <testcase classname="zapcg.Capillary.EarnPointPageTestCases.EarnPoint_Valid_TestCases" name="verifyEarnPointsNavigationFromMeberDetailsScreen" time="7.823"/>
  <testcase classname="zapcg.Capillary.EarnPointPageTestCases.EarnPoint_Valid_TestCases" name="verifyMemberDetailsContentDisplayingOnEarnPointScreen" time="7.865"/>
  <testcase classname="zapcg.Capillary.EarnPointPageTestCases.EarnPoint_Valid_TestCases" name="verifyHeaderHyperlink" time="8.348"/>
  <testcase classname="zapcg.Capillary.EarnPointPageTestCases.EarnPoint_Valid_TestCases" name="verifyBackButtonNavigation" time="8.481"/>
  <testcase classname="zapcg.Capillary.EarnPointPageTestCases.EarnPoint_Valid_TestCases" name="verifyHamBurgerMemberLookupOption" time="9.735"/>
  <testcase classname="zapcg.Capillary.EarnPointPageTestCases.EarnPoint_Valid_TestCases" name="verifyHamBurgerLocationOption" time="9.792"/>
  <testcase classname="zapcg.Capillary.EarnPointPageTestCases.EarnPoint_Valid_TestCases" name="verifyHamBurgerLogoutOption" time="9.382"/>
  <testcase classname="zapcg.Capillary.EarnPointPageTestCases.EarnPoint_Valid_TestCases" name="verifyDateEqualToSystemCurrentDate" time="8.167"/>
  <testcase classname="zapcg.Capillary.EarnPointPageTestCases.EarnPoint_Valid_TestCases" name="verifyTimeEqualToSystemCurrentTime" time="8.465"/>
  <testcase classname="zapcg.Capillary.EarnPointPageTestCases.EarnPoint_Valid_TestCases" name="verifyTransactionAmountShouldNotBeEmpty" time="7.734"/>
  <testcase classname="zapcg.Capillary.EarnPointPageTestCases.EarnPoint_Valid_TestCases" name="emptyTxnAmountButGoToPassPointValue" time="8.120"/>
  <testcase classname="zapcg.Capillary.EarnPointPageTestCases.EarnPoint_Valid_TestCases" name="verifyEligibleEarningPoints" time="8.041"/>
  <testcase classname="zapcg.Capillary.EarnPointPageTestCases.EarnPoint_Valid_TestCases" name="verifySuccessMessageForOnlyTransactionAmount" time="10.876"/>
  <testcase classname="zapcg.Capillary.EarnPointPageTestCases.EarnPoint_Valid_TestCases" name="verifySuccessMessageForRedeemPoint" time="68.072">
    <failure message="An error occurred while verifying the success message: Expected condition failed: waiting for visibility of Proxy element for: DefaultElementLocator &#039;By.xpath: //p[@id=&#039;EPMDLMSG&#039; and contains(text(), &#039;transaction has been successfully submitted. They have earned 35 points and have spent 1 points.&#039;)]&#039; (tried for 10 second(s) with 500 milliseconds interval)" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: An error occurred while verifying the success message: Expected condition failed: waiting for visibility of Proxy element for: DefaultElementLocator 'By.xpath: //p[@id='EPMDLMSG' and contains(text(), 'transaction has been successfully submitted. They have earned 35 points and have spent 1 points.')]' (tried for 10 second(s) with 500 milliseconds interval)
at org.testng.Assert.fail(Assert.java:111)
at zapcg.Capillary.PageObject.earnPointsPage.verifySuccessMessageIfRedeemingPoints(earnPointsPage.java:765)
at zapcg.Capillary.EarnPointPageTestCases.EarnPoint_Valid_TestCases.verifySuccessMessageForRedeemPoint(EarnPoint_Valid_TestCases.java:232)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:64)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:564)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:686)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:230)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:992)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:739)
at org.testng.TestRunner.run(TestRunner.java:614)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:421)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:463)
at org.testng.internal.thread.ThreadUtil.lambda$execute$0(ThreadUtil.java:64)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
at java.base/java.lang.Thread.run(Thread.java:832)
]]>
    </failure>
  </testcase> <!-- verifySuccessMessageForRedeemPoint -->
  <testcase classname="zapcg.Capillary.EarnPointPageTestCases.EarnPoint_Valid_TestCases" name="verifyTaxAssumedAmount" time="8.410"/>
  <testcase classname="zapcg.Capillary.EarnPointPageTestCases.EarnPoint_Valid_TestCases" name="verifyExceedThresoldSuccessMessage" time="9.590">
    <error message="Cannot invoke &quot;org.openqa.selenium.SearchContext.findElement(org.openqa.selenium.By)&quot; because &quot;this.searchContext&quot; is null" type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException: Cannot invoke "org.openqa.selenium.SearchContext.findElement(org.openqa.selenium.By)" because "this.searchContext" is null
at org.openqa.selenium.support.pagefactory.DefaultElementLocator.findElement(DefaultElementLocator.java:68)
at org.openqa.selenium.support.pagefactory.internal.LocatingElementHandler.invoke(LocatingElementHandler.java:38)
at com.sun.proxy.$Proxy18.sendKeys(Unknown Source)
at zapcg.Capillary.PageObject.memberLookupPage.enterMemberId(memberLookupPage.java:94)
at zapcg.Capillary.EarnPointPageTestCases.EarnPoint_Valid_TestCases.verifyExceedThresoldSuccessMessage(EarnPoint_Valid_TestCases.java:288)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:64)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:564)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:686)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:230)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:992)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:739)
at org.testng.TestRunner.run(TestRunner.java:614)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:421)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:463)
at org.testng.internal.thread.ThreadUtil.lambda$execute$0(ThreadUtil.java:64)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
at java.base/java.lang.Thread.run(Thread.java:832)
]]>
    </error>
  </testcase> <!-- verifyExceedThresoldSuccessMessage -->
</testsuite> <!-- zapcg.Capillary.EarnPointPageTestCases.EarnPoint_Valid_TestCases -->
